stix/idl/processing/imaging/stx_plot_fit_map.pro

```diff
@@ -0,0 +1,176 @@
+;+
+;
+; NAME:
+;
+;   stx_plot_fit_map
+;
+; PURPOSE:
+;
+;   Read a STIX science L1 fits file (and potentially a STIX background L1 fits file) and contruct a
+;   calibrated 'stx_visibility' structure
```

Comment:
Seems different to what the function does?

Reply:
Yes, thank, I'll correct this

Reply:
So will we open an issue for this or will it be fixed in this PR?